src/nomad_simulations/schema_packages/properties/thermodynamics.py

```diff
@@ -235,3 +236,31 @@ class Hessian(PhysicalProperty):
         description="""
         """,
     )
+
+    n_negative_eigenvalues = Quantity(
+        type=positive_int(),
+        description="""
+        Number of negative Hessian eigenvalues (imaginary vibrational frequencies).
+        A value of 0 indicates a local minimum, 1 a first-order saddle point, and
+        >1 a higher-order saddle point. Leave unset if the Hessian was evaluated
+        away from a stationary point.
+        """,
+    )
+
+    stationary_point_type = Quantity(
+        type=MEnum(
+            'minimum',
+            'saddle_point',
+            'maximum',
+            'non_stationary',
+        ),
```

Comment:
```suggestion
    stationary_point_type = Quantity(
        type=MEnum(
            'minimum',
            'maximum',
            'saddle_point',
            'non_stationary',
            'soft mode',
            'unavailable',
        ),
```

Not sure whether soft modes are good fit here. They can be distinguished in the Hessian. They mark barrier-less phase transitions and are characterized by vanishing (from the positive side) frequencies.

Reply:
i agree that they are not a good fit here

Soft mode isn’t a stationary‑point class, it’s a numerical/physical symptom (near‑zero curvature) that can appear in any of the actual classes. so for that, i would need to add a new class that conflates numerical symptoms with the true stationary type

Reply:
I agree, the soft mode aligns with a momentum vector, not a single point on the PES. I will add an issue to capture them, but address it next year.
I originally floated them here because the Hessian is one way to first locate them, but they need multiple checks to be ascertained, e.g. checking the MEP.

Reply:
sounds good

Reply:
created. note that my suggestion included more that just `soft mode`

Reply:
added 'unavailable'